test/Common.FunctionalTests/Inprocess/LoggingTests.cs

```diff
@@ -18,48 +17,43 @@ namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
     public class LoggingTests : IISFunctionalTestBase
     {
         private readonly PublishedSitesFixture _fixture;
+        private readonly string _logFolderPath;
 
         public LoggingTests(PublishedSitesFixture fixture)
         {
+            _logFolderPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
             _fixture = fixture;
         }
 
+        public override void Dispose()
+        {
+            base.Dispose();
+            if (Directory.Exists(_logFolderPath))
+            {
+                Directory.Delete(_logFolderPath, true);
+            }
+        }
+
         [ConditionalTheory]
         [InlineData("CheckErrLogFile")]
         [InlineData("CheckLogFile")]
         public async Task CheckStdoutLoggingToFile(string path)
         {
             var deploymentParameters = _fixture.GetBaseDeploymentParameters(publish: true);
 
-            deploymentParameters.WebConfigActionList.Add(
-                WebConfigHelpers.AddOrModifyAspNetCoreSection("stdoutLogEnabled", "true"));
-
-            var pathToLogs = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
-            deploymentParameters.WebConfigActionList.Add(
-                WebConfigHelpers.AddOrModifyAspNetCoreSection("stdoutLogFile", Path.Combine(pathToLogs, "std")));
+            deploymentParameters.EnableLogging(_logFolderPath);
 
             var deploymentResult = await DeployAsync(deploymentParameters);
 
-            try
-            {
-                await Helpers.AssertStarts(deploymentResult, path);
+            await Helpers.AssertStarts(deploymentResult, path);
 
-                StopServer();
+            StopServer();
 
-                var fileInDirectory = Directory.GetFiles(pathToLogs).Single(fileName => fileName.Contains("inprocess"));
-                var contents = File.ReadAllText(fileInDirectory);
+            var fileInDirectory = Directory.GetFiles(_logFolderPath).Single();
```

Comment:
Replace all `Directory.GetFiles(_logFolderPath).Single();` calls with a helper method that uses exact file name